cmd/kubeless/autoscaleCreate.go

```diff
@@ -19,14 +19,14 @@ var autoscaleCreateCmd = &cobra.Command{
 		min, err := cmd.Flags().GetInt32("min")
 		if err != nil {
 			logrus.Fatal(err)
-		} else if min == 0 {
-			logrus.Fatalf("--min can't be 0")
+		} else if min < 0 {
+			logrus.Fatalf("min can't be negative")
```

Comment:
min also can't be zero.  `else if min <= 0`

Reply:
Yeah I noticed that but I think there is no reason to limit `--min=0` 😃 Am I doing wrong ?

Reply:
I don't understand - won't --min=0 end up producing an error from k8s?